.github/renovate.json

```
@@ -0,0 +1,20 @@
+{
+    "extends": ["config:base", ":pinOnlyDevDependencies"],
+    "minimumReleaseAge": "7 days",
+    "automerge": false,
+    "automergeType": "pr",
+    "schedule": ["before 5am on Monday"],
+    "packageRules": [
+      {
+        "matchUpdateTypes": ["minor", "patch"],
+        "automerge": true
```

I think just to be safe we should not automerge anything for the time being. I have a hunch/worry there a lot of unknown unknowns.

```suggestion
        "automerge": false
```

+1 from me in general on automerging non-major updates for stable repos with lots of coverage. IMO that would be a good thing to strive towards as we get ramped up.

---

Seems a good reason, many unknown unknowns. I will create a separate PR with the option to true so we don't forget to revisit this again in few weeks :)